ERCS/eip-7281.md

```diff
@@ -0,0 +1,362 @@
+---
+eip: 7281
+title: Sovereign Bridged Token
+description: An interface for creating fungible representations of tokens bridged across domains.
+author: Shaito (@0xShaito), Excalibor (@excaliborr), Arjun Bhuptani (@arjunbhuptani)
+discussions-to: https://ethereum-magicians.org/t/erc-7281-sovereign-bridged-tokens/14979
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-27
+requires: 20
+---
+
+## Abstract
+
+This proposal defines a minimal extension to [ERC-20](../EIPS/eip-20.md) (affectionately called xERC-20) that enables bridging tokens across domains without creating multiple infungible representations of the same underlying asset. Token issuers can define which bridges are able to mint or burn the xERC-20 on a given domain and configure rate limits for each allowed bridge.
```

Comment:
The bots will complain a lot about "xERC-20" just because of how close it is to a valid "ERC-XXXX" identifier. You could go with `XERC20` as your interface name, then use \`XERC20\` (note the backticks) when referring to it.